ixmp/backend/base.py

```diff
@@ -53,6 +54,42 @@ def get_auth(self, user, models, kind):
         """
         return {model: True for model in models}
 
+    @abstractmethod
+    def add_model(self, name: str):
+        """ Add (register) new model name.
```

Comment:
Please format these correctly:

- No space after the opening triple quote.
- The format for parameters is:
  ```
  name : type
      Description intended four spaces, capitalized, ending with a period.
  ```

See the other methods in this file, and the [numpydoc docstring format](https://numpydoc.readthedocs.io/en/latest/format.html), which is linked from our [style guide](https://docs.messageix.org/en/stable/contributing.html#code-style). Please be sure to compile the docs locally to check that the output is correctly formatted.

Reply:
it looks it @fonfon updated it 84ffa4aa139db5cb0da8d306ad841a06b57432de

Reply:
The change was done after the comment of Paul.
I noticed two more minor leftovers that I'm cleaning up now (release notes and in tests) so I'll add one more commit.